mimesis/builtins/pl.py

```diff
@@ -0,0 +1,100 @@
+"""Specific data provider for Poland (pl)."""
+
+from calendar import isleap
+from typing import Optional
+
+from mimesis.builtins.base import BaseSpecProvider
+from mimesis.enums import Gender
+from mimesis.typing import DateTime
+
+__all__ = ['PolandSpecProvider']
+
+
+class PolandSpecProvider(BaseSpecProvider):
+    """Class that provides special data for Poland (pl)."""
+
+    class Meta:
+        """The name of the provider."""
+
+        name = 'poland_provider'
+
+    def nip(self) -> str:
+        """Generate random valid 10-digit NIP.
+
+        :return: valid 10-digit NIP
+        """
+        nip_digits = list(map(int, str(self.random.randint(101, 998))))
```

Comment:
I'm not sure that I understand why to do it like that. Can you, please, explain to me why not:

```python
[_ for _ in self.random.randint(101, 998)]
```